apps/camera_pipe/fcam/Demosaic.cpp

```diff
@@ -134,54 +93,17 @@ void demosaic(Halide::Tools::Image<uint16_t> input, Halide::Tools::Image<uint8_t
     outHeight /= BLOCK_HEIGHT;
     outHeight *= BLOCK_HEIGHT;
 
-    int WIDTH = outWidth, HEIGHT = outHeight;
-    //fprintf(stderr, "Got outWidth=%d, outHeight=%d\n", outWidth, outHeight);
-
-#if 0
-    Image out(outWidth, outHeight, RGB24);
-
-    // Check we're the right size, if not, crop center
-    if (((input.width() - 8) != (unsigned)outWidth) ||
-        ((input.height() - 8) != (unsigned)outHeight)) {
-        int offX = (input.width() - 8 - outWidth)/2;
-        int offY = (input.height() - 8 - outHeight)/2;
-        offX -= offX&1;
-        offY -= offY&1;
-
-        if (offX || offY) {
-            input = input.subImage(offX, offY, Size(outWidth+8, outHeight+8));
-        }
-    }
-#endif
-
     // Prepare the lookup table
-    unsigned char lut[4096];
+    unsigned char lut[1024];
     makeLUT(contrast, blackLevel, gamma, lut);
 
     // Grab the color matrix
     float colorMatrix[12];
-    #if 0
-    // Check if there's a custom color matrix
-    if (src.shot().colorMatrix().size() == 12) {
-        for (int i = 0; i < 12; i++) {
-            colorMatrix[i] = src.shot().colorMatrix()[i];
-        }
-    } else {
-        // Otherwise use the platform version
-        src.platform().rawToRGBColorMatrix(src.shot().whiteBalance, colorMatrix);
-    }
-    #else
     makeColorMatrix(colorMatrix, colorTemp);
-    #endif
 
-#if 0
-    for (int by = 0; by < rawHeight-8-BLOCK_HEIGHT+1; by += BLOCK_HEIGHT) {
-        for (int bx = 0; bx < rawWidth-8-BLOCK_WIDTH+1; bx += BLOCK_WIDTH) {
-#else
-    //#pragma omp parallel for
+    #pragma omp parallel for
```

Comment:
Yes, although it may have been a bad idea, the reason the CI build broke was because OpenMP libs are missing.